### ISSUE TITLE
microservice-tutorial docker-image

### ISSUE THREAD

**Issue by bradmennengaHK** [dotnet/core#7168]
Problem encountered on https://dotnet.microsoft.com/en-us/learn/aspnet/microservice-tutorial/docker-image
Operating System: windows

Provide details about the problem you're experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.

https://dotnet.microsoft.com/en-us/learn/aspnet/microservice-tutorial/docker-image

docker build -t mymicroservice .
[+] Building 0.4s (9/14)
 => [internal] load build definition from Dockerfile                                                                                                           0.1s
 => => transferring dockerfile: 329B                                                                                                                           0.0s
 => [internal] load .dockerignore                                                                                                                              0.1s
 => => transferring context: 70B                                                                                                                               0.0s
 => [internal] load metadata for mcr.microsoft.com/dotnet/aspnet:6.0                                                                                           0.2s
 => [internal] load metadata for mcr.microsoft.com/dotnet/sdk:6.0                                                                                              0.2s
 => [build 1/6] FROM mcr.microsoft.com/dotnet/sdk:6.0@sha256:a3dd4dee05cd1369014244d03b28b602e6a2e1650210dd8633322e00379471ec                                  0.0s
 => [stage-1 1/3] FROM mcr.microsoft.com/dotnet/aspnet:6.0@sha256:545da8027d0de13ac88716fc380201f05ad9b4834b5b2fd95eadf277e51318a7                             0.0s
 => [internal] load build context                                                                                                                              0.0s
 => => transferring context: 3.68kB                                                                                                                            0.0s
 => CACHED [build 2/6] WORKDIR /src                                                                                                                            0.0s
 => ERROR [build 3/6] COPY MyMicroservice.csproj .                                                                                                             0.0s
------
 > [build 3/6] COPY MyMicroservice.csproj .:
------
failed to compute cache key: "/MyMicroservice.csproj" not found: not found

**Comment by olaffveregscan** [comment]
Did you fixed?

**Comment by bradmennengaHK** [comment]
Yes, I was able to get it to work..  Issue was the project could not be located..   PATH environment variable needed to be updated to include new folder location for test.
